src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -61,17 +60,58 @@ std::string getEntities
   ParseData*                 parseDataP
 )
 {
-  std::string  answer;
-  Entities     entities;
+  using namespace std;
+  typedef vector<string>  vstr;
+  typedef vstr::size_type vstr_sz;
+
+  Entities entities;
+  string   answer;
+
+  // optional parameter for list of IDs
+  string paramIDs     = ciP->uriParam["id"];
+  string paramPattern = ciP->uriParam["idPattern"];
+  string pattern      = ".*"; // all entities, default value
 
+  if (paramPattern != "" && paramIDs != "")
+  {
+    ciP->httpStatusCode = SccBadRequest;
+    answer = "{\"error\":\"Incompatible parameters: id, IdPattern\"}";
```

Comment:
I though you refer to render `orionError` literarly in the reponse JSON (as we do in v1). However, if you refer to use the internal class OrionError for that *as long as* it renders correctly following the NGSIv2 specification, its fine.